### DISCUSSION TITLE
Using CCA to integrate scRNA-seq datasets, but Harmony to integrate scATAC-seq datasets before label transfering

### DISCUSSION THREAD

**Discussion by RegnerM2015** [stuart-lab/signac#734] [Q&A] [answered]
Continuing our discussion from https://github.com/timoast/signac/discussions/587 @timoast,

Considering the following hypothetical scenario:

The goal is to transfer labels between matched scRNA-seq and scATAC-seq datasets for a total of 10 tissue samples.

1) We combine the 10 scRNA-seq datasets using CCA integration to create one multi-sample scRNA-seq dataset and recluster the dataset with Louvain clustering.

 2) We combine the 10 scATAC-seq datasets using Harmony integration to create one multi-sample scATAC-seq dataset.

 3) Finally, we use CCA to transfer labels from the multi-sample scRNA-seq dataset to the multi-sample scATAC-seq dataset.

Are these steps and strategies reasonable? Is it okay to use CCA batch correction in scRNA-seq but Harmony integration in scATAC-seq?
Or would it be preferable to use Harmony integration for both scRNA-seq and scATAC-seq before transfering labels?

Thanks for your help!

**Comment by timoast** [comment, marked as answer]
https://github.com/timoast/signac/discussions/587#discussioncomment-1039751
